homeassistant/components/auth/indieauth.py

```diff
@@ -92,7 +92,7 @@ async def fetch_redirect_uris(hass: HomeAssistant, url: str) -> list[str]:
     parser = LinkTagParser("redirect_uri")
     chunks = 0
     try:
-        async with aiohttp.ClientSession() as session:
+        async with aiohttp.ClientSession(trust_env=True) as session:
```

Comment:
Your PR description does not match with changing this auth code.

Reply:
Hello! Thanks for your review. I have updated PR description. I think HomeAssistant and third-party integrations should respect the user's proxy settings, so I added `trust_env=True` wherever aiohttp is used.